guides/source/security.md

```diff
@@ -50,9 +50,9 @@ User.find(session[:user_id])
 
 ### Session id
 
-NOTE: _The session id is a 32 byte long MD5 hash value._
+NOTE: _The session id is a 32-character random hex string._
```

Comment:
The code:

```
def generate_sid
    sid = SecureRandom.hex(16)
    sid.encode!(Encoding::UTF_8)
    sid
end
```

The line `SecureRandom.hex(16)` generates 32-character string as per documentation: 

[SecureRandom::hex](http://ruby-doc.org/stdlib-2.2.1/libdoc/securerandom/rdoc/SecureRandom.html#method-c-hex)

The argument n specifies the length, in bytes, of the random number to be generated. The length of the resulting **hexadecimal string is twice n**.